core/src/main/java/org/mule/api/context/notification/ServerNotification.java

```diff
@@ -264,4 +266,27 @@ private static Object verifyNullResource(Object resource)
         return resource;
     }
 
+    private final class LazyEventName implements Serializable
+    {
+        private static final long serialVersionUID = 5162290824647627016L;
+
+        private volatile String eventClassName;
+
+        @Override
+        public String toString()
+        {
+            if(eventClassName == null)
+            {
+                synchronized (this)
+                {
+                    if(eventClassName == null)
+                    {
+                        eventClassName = ServerNotification.this.getClass().getSimpleName();
```

Comment:
Would be the same value to do this instead of `ClassUtils.getClassName(getClass())`?